onionmsg/onion.go

```diff
@@ -68,22 +68,41 @@ func createPathToBlind(path []*btcec.PublicKey,
 
 // blindedToSphinx converts the blinded path provided to a sphinx path that can
 // be wrapped up in an onion.
-func blindedToSphinx(blindedRoute *sphinx.BlindedPath) (*sphinx.PaymentPath,
-	error) {
+func blindedToSphinx(blindedRoute *sphinx.BlindedPath,
```

Comment:
Update comment to reflect that we also tlv-stream-ify the final payload + encrypted data